src/inet_address.rs

```diff
@@ -0,0 +1,40 @@
+use SocketFamily;
+use ffi;
+use glib::translate::*;
+use InetAddress;
+
+#[derive(Debug)]
+pub enum InetAddressOctet<'a> {
```

Comment:
The name does not seem ideal. It's not a single octet (== byte) but 4 or 16 bytes. Maybe instead of using the rare word octet, we could call this `InetAddressBytes` instead?

Reply:
Okay, let me rename :-), I used `Octet` to match `ip.octets()`.

Reply:
@EPashkin @GuillaumeGomez do you have suggestions? I would call it bytes because that's how it's called in GIO :)

Reply:
IMHO `Octets` or `Bytes` is almost same, `Octet` is really not.
https://en.wikipedia.org/wiki/IP_address uses "group of 8 bits (an octet)" but not `byte` maybe because byte can have different size.
So I for `InetAddressOctets`

Reply:
In french, octet is the translation of byte so it seemed good to me. :3

For the name, `InetAddressData` might be enough?

Reply:
@EPashkin That was true decades ago, nowadays a byte is always 8 bit fortunately. And the GIO API talks about bytes here (`new_from_bytes()` for example), not octets.

Reply:
Ok, then `Bytes`, just `Data` IMHO not clear

Reply:
I had force-pushed with InetAddressBytes :-).